crates/matrix-sdk-crypto/src/verification/requests.rs

```diff
@@ -366,8 +385,22 @@ impl VerificationRequest {
     /// based verification.
     #[cfg(feature = "qrcode")]
     pub async fn generate_qr_code(&self) -> Result<Option<QrVerification>, CryptoStoreError> {
-        let inner = self.inner.get();
-        inner.generate_qr_code(self.we_started, self.inner.clone().into()).await
+        let inner = self.inner.get().to_owned();
```

Comment:
is the `to_owned` meaningful? `get()` already returns a clone of the observed value.

Reply:
You are correct, the explicit `drop` calls become useless as well. Thanks for noticing.